apps/molgenis-components/src/components/tables/RoutedTableExplorer.vue

```diff
@@ -0,0 +1,223 @@
+<template>
+  <div>
+    <TableExplorer
+      :tableName="tableName"
+      :graphqlURL="graphqlURL"
+      :canEdit="canEdit"
+      :canManage="canManage"
+      @update:conditions="updateConditions"
+      @update:showColumns="updateColumns"
+      @update:showFilters="updateFilters"
+      @update:showPage="updatePage"
+      @update:showLimit="updateLimit"
+      @update:showOrder="updateOrder"
+      @update:showView="updateView"
+      :showView="getView()"
+      :showColumns="getColumns()"
+      :showFilters="getFilters()"
+      :conditions="getConditions()"
+      :showPage="getPage()"
+      :showLimit="getLimit()"
+      :showOrderBy="getOrderBy()"
+      :showOrder="getOrder()"
```

Comment:
what happend to the idea of combining these into a single object

why the use of ':' in the name ? , rest of the events use camelCase

Reply:
I did not wat to change the existing component interface. But i think it's not used a lot in other places, we probably could change it.

Reply:
I don't want to do that in this PR. I'm still all for it, but it's already quite big.

Reply:
The ':' is because copy paste from old code.

Reply:
check